modules/gapi/perf/common/gapi_render_perf_tests.hpp

```diff
@@ -0,0 +1,33 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_RENDER_PERF_TESTS_HPP
+#define OPENCV_GAPI_RENDER_PERF_TESTS_HPP
+
+
+
+#include "../../test/common/gapi_tests_common.hpp"
+#include <opencv2/gapi/imgproc.hpp>
+
+namespace opencv_test
+{
+
+  using namespace perf;
+
+  //------------------------------------------------------------------------------
+
+    class RenderTestTexts : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
+    class RenderTestRects : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
+    class RenderTestCircles : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
+    class RenderTestLines : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
+    class RenderTestMosaics : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
+    class RenderTestImages : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
+    class RenderTestPolylines : public TestPerfParams<tuple<compare_f, MatType, cv::Size, cv::GCompileArgs>> {};
```

Comment:
Could you add to test with multiple primitives.
For example:
n - `Rect`
n - `Mosaic` inside of `Rects`
m - `Text`
k - `Image`

n, m, k - number of each primitive (`Hint`: you can make it as a test parameters )

Reply:
I did this test, but now it is red in the case of a 640x480 matrix. I'm working on it.
But you can look at this test, did I understand you correctly?

Reply:
This test is done now. 

But!
Test framework allows to specify only 10 test parameters
Therefore, I cannot specify all the parameters for the image, text, mosaic, rectangle in the test instance.
All these parameters are announced in the body of the test.